Cloud-Engineering/L4 - Lead Cloud Engineer.md

```diff
@@ -0,0 +1,75 @@
+# L4: Lead Cloud Engineer
+
+> _I am a pivotal member of the Octopus team, balancing the responsibilities of delivering high-quality projects and guiding technical initiatives. I bridge the gap between engineering and architecture, providing leadership, technical expertise, and a focus on reliability and security to drive projects forward._
+
+**Planning horizon:**  Next cycle to 6 months  
+**Impact radius:** Team (4-10)  
+**Evaluation:** Manager  
+**Responsibility and direction needed:** Takes ownership of complex projects and leads them from idea to production with moderate supervision.
+
+## 🦉 Domain Expertise
+
+- I demonstrate solid experience with server operating systems, network, and security fundamentals.
+- I am comfortable exploring moderately risky situations with limited upfront information.
+- I stay updated with evolving standards, platform features, critical technologies, and security best practices from vendors and the community.
+- I can design and implement infrastructure as code and guide my team in these practices.
+- I am proficient with an array of cloud technologies (virtualization, containerization, serverless, virtual networking), making informed decisions on scalability, reliability, and maintainability.
+- I identify and address technical debt in line with team priorities.
+- I have a good understanding of .NET fundamentals and contribute efficiently to the Octopus Cloud codebase (equivalent to at least [L3 Senior Software Engineer](../Software-Engineering/L3-Senior-Software-Engineer.md)).
+- I make informed architectural decisions considering cross-cutting concerns such as infrastructure, security, scalability, and maintainability.
+- I raise potential problems with my team before getting blocked, ask for help when needed, and offer help when I can assist.
+- I integrate reliability engineering principles into my work, focusing on automation, observability, and efficient incident response.
```

Comment:
Minor nit - should we use "effective" vs "efficient"?

In my mind being efficient is important, but ultimately what we want is effectiveness - i.e. outcomes vs process.